### PR TITLE
feat: [FFM-4353]: global style bug fix

### PR DESCRIPTION
**Pull request by crystal-fong**
### Summary
CSS tags applied to the tag affected `Allowed Values` component within Pipelines. 
This PR limits this change within the Tags within Services modal

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 #### 1 Issues 
* Code issue 1 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10589)

**Comment by crystal-fong** [comment]
retrigger titlecheck

**Review by chrisblakely01 — APPROVED** [review]

**Review by upadhyay-an — APPROVED** [review]

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good

File: src/modules/75-cf/components/FlagActivation/ServicesList.module.scss [hunk 1]
```diff
@@ -16,10 +16,12 @@
   color: var(--grey-900);
   background: var(--grey-50);
 }
-:global {
-  .bp3-active,
-  .bp3-tag:hover {
-    background: var(--primary-6) !important;
-    color: var(--grey-0) !important;
+.servicesListModal {
```
Comment (mtran7) [inline src/modules/75-cf/components/FlagActivation/ServicesList.module.scss:19]: Usually we avoid using global and use   ['className*="serviceListModal"']

**Pull request merged**